test/verify/check-realms

```diff
@@ -236,7 +236,8 @@ class TestRealms(MachineCase):
         b.click("#system-info-domain a")
         b.wait_popup("realms-op")
         b.set_val(".realms-op-address", "NOPE")
-        b.wait_js_cond("ph_find('.realms-op-address-error').title != ''")
+        b.wait_visible(".realms-op-address-error")
+        b.wait_attr(".realms-op-address-error", "title", "Domain NOPE could not be contacted")
```

Comment:
I don't understand why hovering the mouse over the span by hand triggers shows this tooltip but with the b.mouse over the span nothing gets shown. Could you please give me some hint?

Because with this line here we don't check if the tooltip is visible. This exists even if we don't hover over the explanation mark to show the string.

Reply:
`title=` is *entirely* implemented in browsers. The tooltip you see is not reflected at all in the DOM. The bootstrap tooltips OTOH are proper HTML, so they can be tested in the DOM. So testing that the element is visible and has the expected title is sufficient to validate the tooltip.

As I said, we should reactify the tooltip and use the bootstrap one, but that can happen in a follow-up.